cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -170,7 +170,41 @@ def test_no_error_while_fetching_the_service():
             1.
             2. In SUI click on provisioned service
     """
-    pass
+    user, _ = user_self_service_role
+
+    # login with user having self service role
+    with user:
+        with appliance.context.use(ViaUI):
+            appliance.server.login(user)
```

Comment:
This is actually redundant, the user context manager handles this.

https://github.com/ManageIQ/integration_tests/blob/master/cfme/configure/access_control/__init__.py#L158